subprojects/docs/src/docs/userguide/java_testing.adoc

```diff
@@ -54,13 +54,11 @@ It's also worth bearing in mind that the `test` source set automatically creates
 
 All you need to do in most cases is configure the appropriate compilation and runtime dependencies and add any necessary configuration to the `test` task. The following example shows a simple setup that uses JUnit 4.x and changes the maximum heap size for the tests' JVM to 1 gigabyte:
```

Comment:
I think we missed a few samples in this chapter: "filtering" and "unit test report" and a few others. Perhaps this PR is still WIP.

Reply:
Scratch that. I hadn't seen PR #6518 yet.